sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -52,12 +52,17 @@ export function createSpanFunction(args: CreateSpanFunctionArgs) {
       ...tracingOptions.spanOptions
     };
 
-    const span = tracer.startSpan(`${args.packagePrefix}.${operationName}`, spanOptions);
+    const spanName =
+      args.packagePrefix !== undefined ? `${args.packagePrefix}.${operationName}` : operationName;
+    const span = tracer.startSpan(spanName, spanOptions);
 
-    span.setAttribute("az.namespace", args.namespace);
+    if (args.namespace !== undefined) {
```

Comment:
empty string would also make these checks simpler, e.g.

```suggestion
    if (args.namespace) {
```

Reply:
Agreed :)